external/NETNative/project.json.template

```diff
@@ -0,0 +1,12 @@
+﻿{
+  "frameworks": {
+    "{TFM}": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Private.NETNative": "1.1.0-beta-24911-00"
+      }
+    }
+  },
+  "runtimes": {
+    "{RID}": {}
```

Comment:
If this isn't RID specific or you don't need a variable TFM you don't need to use a template, although it doesn't really hurt anything.